pkg/driver/bacnet/merge/emergency.go

```diff
@@ -21,12 +21,15 @@ import (
 	"github.com/vanti-dev/sc-bos/pkg/task"
 )
 
-// AlarmConfig allows configuring a specific bacnet point to raise an Emergency if the
-// value read from that point is anything other than OkValue
+// AlarmConfig allows configuring a specific bacnet point to raise an Emergency if either:
+// - value read from that point is anything other than OkValue
+// - value read from that point is less than GreaterThanOkValue
 type AlarmConfig struct {
 	config.ValueSource
-	OkValue     int    `json:"okValue"`     // what we expect to read from the point when it is ok, any other value is an emergency
-	AlarmReason string `json:"alarmReason"` // the reason of the alarm
+	OkValue            int    `json:"okValue"`            // what we expect to read from the point when it is ok, any other value is an emergency. Not supported for float values.
+	GreaterThanOkValue int    `json:"greaterThanOkValue"` // we expect the point to be greater than this value, if it isn't we have an emergency
+	AlarmReason        string `json:"alarmReason"`        // the reason of the alarm
+	ValueType          string `json:"valueType"`          // the type of the value we are expecting to read, int, float etc.
```

Comment:
We should be able to do some type checking on the response without the user needing to configure this. BACnet is already typesafe in its transport and that filters through the gobacnet library. What I'm saying is that the response will be a `float32` or `int32` by the time we see it so you can check for this instead of configuring it.